### ISSUE TITLE
Create UX design of focus navigation in JupyterLab

### ISSUE THREAD

**Issue by clapierre** [jupyterlab/jupyterlab#6578]
From Web4All Hackathon https://github.com/diagram-codesprint/jupyterlab/issues/14
**Describe the issue**
The focus navigation using the keyboard only is not accessible.  The focus is not identifiable nor predictable in its behavior.

**To Do**
Create the design document.

**Comment by isabela-pf** [comment]
This is covered more granularly in #9399, so I'm closing this issue in favor of the larger review.
